mode/threads.py

```diff
@@ -184,7 +186,10 @@ async def _keepalive2(self) -> None:
                 if monotonic() - self.last_wakeup_at > 3.0:
                     self.log.error("Thread keepalive is not responding...")
             asyncio.run_coroutine_threadsafe(
```

Comment:
try simply use:

```
await self._wakeup_timer_in_thread()
```

No `asyncio.run_coroutine_threadsafe` and `asyncio.ensure_future`.

`run_coroutine_threadsafe` is not required because this is runed under current thread.

Reply:
```
await self.sleep(0)
asyncio.run_coroutine_threadsafe(asyncio.sleep(0), self.parent_loop)
```

Both `sleep` are necessary. The first runs in the `thread_loop` and the second runs in the `parent_loop`